
From: Tom Rini <trini@kernel.crashing.org>

<linux/fd.h> references __user which is defined in <linux/compiler.h>.

Signed-off-by: Tom Rini <trini@kernel.crashing.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/fd.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/linux/fd.h~add-linux-compilerh-to-linux-fdh include/linux/fd.h
--- 25/include/linux/fd.h~add-linux-compilerh-to-linux-fdh	2004-06-28 17:42:24.133545600 -0700
+++ 25-akpm/include/linux/fd.h	2004-06-28 17:42:24.137544992 -0700
@@ -2,6 +2,7 @@
 #define _LINUX_FD_H
 
 #include <linux/ioctl.h>
+#include <linux/compiler.h>
 
 /* New file layout: Now the ioctl definitions immediately follow the
  * definitions of the structures that they use */
_
